src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/ProfessionalUserServiceImplTest.java

```diff
@@ -560,18 +566,13 @@ public void findUserStatusByEmailForPendingOrgThrowsException() throws Exception
     }
 
     @Test(expected = ExternalApiException.class)
-    public void findUserStatusByEmailForActiveThrowsExceptionWhenUpServiceDown() throws Exception {
+    public void findUserStatusByEmailForActiveThrowsExceptionWhenUpServiceDown() {
         organisation.setStatus(OrganisationStatus.ACTIVE);
 
```

Comment:
prefix test and add verify for when conditions

Reply:
this has been addressed in the following commit: https://github.com/hmcts/rd-professional-api/pull/620/commits/b5448bbd468688f02bf1fd93ebc725364a468785